packages/nextjs/test/integration/components/Layout.tsx

```diff
@@ -7,6 +7,9 @@ type Props = {
   title?: string;
 };
 
+// Defined as any so it can also be used in versions of Next.js that do not support the legacyBehavior prop.
+const linkProps: any = { legacyBehavior: true };
+
```

Comment:
Just ran the tests against Next 13 without the `legacyBehavior` prop and it all seemed to work. Just to confirm, I also ran it with a page like this:

```jsx
const HealthyPage = (): JSX.Element => (
  <Link href="/alsoHealthy">
    <a>
      <a>
        <a>
          <a>
            <a id="alsoHealthy">AlsoHealthy</a>
          </a>
        </a>
      </a>
    </a>
  </Link>
);
```

It seems the nested `<a>` tags don't pose a problem, so for simplicity, let's not worry about adding `linkProps` everywhere.

Reply:
For me the integration tests fail when not adding the `legacyBehavior` prop.

Reply:
Maybe it was just a fluke on my machine, because it seems to OK on CI.